engines/sci/graphics/palette.cpp

```diff
@@ -853,7 +853,7 @@ void GfxPalette::palVaryCallback(void *refCon) {
 
 void GfxPalette::palVaryIncreaseSignal() {
 	if (!_palVaryPaused)
-		_palVarySignal++;
```

Comment:
And since this isn't apparent, the comment where I'm asking for changes/commits to be addressed refers to this change